compiler/test/dotty/tools/vulpix/ParallelTesting.scala

```diff
@@ -1077,14 +1077,16 @@ trait ParallelTesting extends RunnerOrchestration { self =>
     /** Extract `Failure` set and render from `Test` */
     private def reasonsForFailure(test: Test): String = {
       val failureReport =
-        if (test.failureCount == 0) ""
-        else s"\n  - encountered ${test.failureCount} test failures(s)"
+        if test.failureCount == 0 then ""
+        else s"encountered ${test.failureCount} test failure(s):\n"
 
       failureReport + test.failureReasons.collect {
         case test.TimeoutFailure(title) =>
           s"  - test '$title' timed out"
         case test.JavaCompilationFailure(msg) =>
           s"  - java compilation failed with:\n${ msg.linesIterator.map("      " + _).mkString("\n") }"
+        case test.Generic =>
```

Comment:
Why is this case needed now?

Reply:
it's just to show generic failures in the output, otherwise they don't appear and it's weird when you have only generic failures in the list